config-generators/mssql-commands.txt

```diff
@@ -188,3 +189,4 @@ update series --config "dab-config.MsSql.json" --permissions "TestNestedFilterMa
 update series --config "dab-config.MsSql.json" --permissions "TestNestedFilterManyOne_EntityReadForbidden:create,update,delete"
 update series --config "dab-config.MsSql.json" --permissions "TestNestedFilterOneMany_ColumnForbidden:read"
 update series --config "dab-config.MsSql.json" --permissions "TestNestedFilterOneMany_EntityReadForbidden:read"
+update DefaultBuiltInFunction --config "dab-config.MsSql.json" --permissions "anonymous:create" --fields.exclude "current_date,next_date"
```

Comment:
Is this a bug? there are no columns in the associated table name `next_date`. I only see `next_day`. Is there an aliasing that occurs somewhere that I missed? Shouldn't DAB error out if you provided an invalid column name?

Reply:
it just ignores, if a column is mentioned which does not exist.